File: src/main/java/com/karkabulary/web/definition/Definition.java
package com.karkabulary.web.definition;

public class Definition {
    public final String term;
    public final String description;

    public Definition(String term, String description) {
        this.term = term;
        this.description = description;
    }

    public String getTerm() {
        return term;
    }

    public String getDescription() {
        return description;
    }
}

File: Karkabulary/Models/Definitions/Definition.cs
namespace Karkabulary.Models.Definitions
{
    public class Definition {
        public string Term { get; private set; }
        public DefinitionType DefinitionType { get; private set; }
        public string Description { get; private set; }

        public Definition(string term, DefinitionType definitionType, string description) {
            Term = term;
            DefinitionType = definitionType;
            Description = description;
        }
    }

    public enum DefinitionType {
        Phrase,
        Thing,
        Action
    }
}
